recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "spdlog" %}
-{% set version = "1.9.2" %}
-{% set sha256 = "6fff9215f5cb81760be4cc16d033526d1080427d236e86d70bb02994f85e3d38" %}
+{% set version = "1.8.5" %}
```

Comment:
Wouldn't this downgrade the version?

Reply:
Yes, because 1.8.5 is missing for osx-arm64. However, what usually is done is to push this PR into a new branch, call it 1.8.x and so you don't have to worry about the current flow.

Alternatively, if it downgrades the version, you could always update the version again, either manually or the bot will figure it out for you. 

Does that make sense?

Reply:
https://conda-forge.org/docs/maintainer/updating_pkgs.html?highlight=multiple#maintaining-several-versions